z3/tests/lib.rs

```diff
@@ -865,6 +993,136 @@ fn test_array_store_select() {
 
 #[test]
 #[should_panic]
+fn test_goal_get_formulas_empty() {
```

Comment:
is there any particular reason not to return an empty vec when the size is 0 instead of panicking?